components/myvac_button.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import FooterLink from "./typography/footer_link";
+import FolderMouse from "./icons/FolderMouse";
+import { css } from "emotion";
+import { globalTheme } from "../theme";
+
+const desktopButton = css`
+  font-size: 10px;
+  text-transform: uppercase;
+  line-height: 23px;
+  letter-spacing: 2.5px;
+  margin-left: 30px;
+  svg {
+    margin-top: 5px;
+    vertical-align: middle;
+    padding-left: 10px;
+    color: ${globalTheme.colour.blueGrey};
+  }
+`;
+
+class MyVacButton extends Component {
+  render() {
+    const { t } = this.props;
+    return (
+      <FooterLink href={t("myvac_register_href")} className={desktopButton}>
```

Comment:
Assuming most of our users are new to VAC, it can make sense to use the register link here, but are we doing this on purpose? The VAC site header uses https://www.veterans.gc.ca/eng/e_services/ for the My VAC Account button.

Reply:
Good catch! I didn't use a different link on purpose.